### ISSUE TITLE
godoc command not found after install

### ISSUE THREAD

**Issue by ruthmoog** [quii/learn-go-with-tests#653]
[in chapter Hello World](https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/hello-world#go-doc), after the install for godoc terminal does now allow the `godoc` command.
Instruction should be updated or removed.

```
ruthmoog hello % go install golang.org/x/tools/cmd/godoc@latest
go: downloading golang.org/x/tools v0.7.0
go: downloading golang.org/x/sys v0.6.0
go: downloading github.com/yuin/goldmark v1.4.13
go: downloading golang.org/x/mod v0.9.0
ruthmoog hello % godoc -http:8000                              
zsh: command not found: godoc
```

**Comment by 0xckylee** [comment]
After googling around, I found out this is issue with $PATH. When you install `go install golang.org/x/tools/cmd/godoc@latest`the binary is installed to (on Mac) /User/[username]/go/bin so you have to set $PATH for this directory.

I directly edited /etc/paths & restarted the terminal, but apparently this is not best practices. Check out below links for better ways & references:
https://forum.golangbridge.org/t/unable-to-install-godoc-on-my-local-machine/30064
https://stackoverflow.com/questions/7703041/editing-path-variable-on-mac
https://medium.com/geekculture/path-macos-best-practice-for-path-environment-variables-on-mac-os-35ec4076a486

I will make a PR to this issue.

**Comment by Battlesheepu** [comment]
Bump,

I've noticed this problem after browsing through docs - official Go instructions do not include any information about adding the `$GOPATH/bin` to user's `$PATH`. Maybe we should add this to the tutorial?

Another, minor thing - it seems `godoc` has been replaced by `pkgsite`, too. Is that worthy of a new issue/MR?

**Comment by microwavestine** [comment]
@Battlesheepu Yup, that's the expected behavior but I wasn't aware of it until I came across this issue. I believe this part in Install Go section of docs was misleading for beginners like me (who never had GOPATH set before):

> This approach is the default build mode since Go 1.16, therefore the use of GOPATH is not recommended.

I haven't followed through tutorial for too long... thanks for raising the issue and keeping it updated.

**Comment by quii** [comment]
Same fix as #691 — commit f3546d8 replaced the godoc instructions with go doc/pkgsite, referencing both this issue and #691 directly. Since readers no longer install/run godoc at all, this shouldn't come up anymore. Closing.
